### PR TITLE
[REF] inventory: valuation section

### PR DESCRIPTION
**Pull request by Felicious**
Move inventory valuation files to the `product_management` folder.

```
🟢 ADD
❌ REMOVE
🔘 RENAME/MOVE

inventory_and_mrp/
└── inventory/
    ├── product_management/
    └──  warehouses_storage
         └── 🔘 inventory_valuation/ --> ../product_management/
              ├── 🔘 integrating_landed_costs.rst --> ../../product_management/inventory_valuation/integrating_landed_costs.rst
              ├── 🔘 inventory_valuation_config.rst --> ../../product_management/inventory_valuation/inventory_valuation_config.rst
              └── 🔘 using_inventory_valuation.rst --> ../../product_management/inventory_valuation/using_inventory_valuation.rst
```

Fwport: yes

### CODE REVIEW

**Comment by Felicious** [comment]
Hi @samueljlieber ! This is *another* request from the PEs to move the files to another folder (:

**Comment by samueljlieber** [comment]
Hi @Felicious! Can you please squash commits before I review, thank you!

File: redirects/16.0.txt [hunk 1]
```diff
@@ -88,8 +88,11 @@ applications/inventory_and_mrp/inventory/routes/concepts/use_routes.rst applicat
...
 applications/inventory_and_mrp/inventory/management/reporting.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation.rst                                                                         # /inventory_and_mrp/inventory/management/reporting -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation
 applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst                   # /inventory_and_mrp/inventory/management/reporting/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst applications/inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config.rst # /inventory_and_mrp/inventory/warehouses_storage/* -> /inventory_and_mrp/inventory/product_management/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:91]: Please update redirect comment to be a little more accurate to the move :)
```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst applications/inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config.rst # /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/* -> /inventory_and_mrp/inventory/product_management/inventory_valuation/*
```

File: redirects/16.0.txt [hunk 2]
```diff
@@ -88,8 +88,11 @@ applications/inventory_and_mrp/inventory/routes/concepts/use_routes.rst applicat
...
 applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/using_inventory_valuation.rst                     # /inventory_and_mrp/inventory/management/reporting/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/using_inventory_valuation.rst applications/inventory_and_mrp/inventory/product_management/inventory_valuation/using_inventory_valuation.rst   # /inventory_and_mrp/inventory/warehouses_storage/* -> /inventory_and_mrp/inventory/product_management/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:93]: This one too please!
```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/using_inventory_valuation.rst applications/inventory_and_mrp/inventory/product_management/inventory_valuation/using_inventory_valuation.rst   # /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/* -> /inventory_and_mrp/inventory/product_management/inventory_valuation/*
```

File: redirects/16.0.txt [hunk 3]
```diff
@@ -88,8 +88,11 @@ applications/inventory_and_mrp/inventory/routes/concepts/use_routes.rst applicat
...
 applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst                       # /inventory_and_mrp/inventory/management/reporting/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst applications/inventory_and_mrp/inventory/product_management/inventory_valuation/integrating_landed_costs.rst     # /inventory_and_mrp/inventory/warehouses_storage/* -> /inventory_and_mrp/inventory/product_management/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:95]: Last one :)
```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst applications/inventory_and_mrp/inventory/product_management/inventory_valuation/integrating_landed_costs.rst     # /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/* -> /inventory_and_mrp/inventory/product_management/inventory_valuation/*
```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, nice work! Just a couple quick fixes for redirect comments :)

Thank you for your work!
..
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**